nginx/fulltest/genconfig.py

```diff
@@ -114,23 +141,27 @@ def write_nginx_config(f, i, cf, port, sig, k):
 
            # deactivate if you don't like tables:
            i.write("<tr><td>"+sig+"</td><td>"+k+"</td><td>"+str(port)+"</td><td><a href=https://"+TESTFQDN+":"+str(port)+">"+sig+"/"+k+"</a></td></tr>\n")
-           if k in chromium_algs and not ("_" in sig and (sig.startswith("p") or (sig.startswith("rsa")))):
-               cf.write("<tr><td>"+sig+"</td><td>"+k+"</td><td>"+str(port)+"</td><td><a href=https://"+TESTFQDN+":"+str(port)+">"+sig+"/"+k+"</a></td></tr>\n")
+           # chromium support discontinued
```

Comment:
Is this correct given https://github.com/open-quantum-safe/oqs-demos/pull/245? @pi-314159 : What's your take? Should we completely remove explicit reference to Chromium?

Reply:
We wanted to not include an explicit support list anymore for Chromium, but I think the statement "# chromium support discontinued" is incorrect with the work by @pi-314159.
I'd be in favor to update the genconfig.py file by removing references to Chromium (particularly that chromium support is discontinued).